xsd/UFTP-agr-dso.xsd

```diff
@@ -259,6 +278,11 @@
                         <xs:documentation>Reference to the concerning contract, if applicable. The contract may be either bilateral or commoditized market contract.</xs:documentation>
                     </xs:annotation>
                 </xs:attribute>
+                <xs:attribute name="ServiceType" type="xs:string" use="optional">
+                    <xs:annotation>
+                        <xs:documentation>Service type for this request, the service type determines response characteristics such as latency or asset participation type.</xs:documentation>
```

Comment:
`Service type for this request. The service type determines response characteristics such as latency and asset participation type.`